universe/node/api_profile.go

```diff
@@ -23,28 +23,25 @@ func (n *Node) apiProfileUpdate(c *gin.Context) {
 	}{}
 
 	if err := c.ShouldBindJSON(&inBody); err != nil {
-		n.log.Debug(errors.WithMessage(err, "Node: apiProfileUpdate: failed to bind json"))
-		c.AbortWithStatusJSON(http.StatusBadRequest, gin.H{
-			"message": "invalid request body",
-		})
+		err = errors.WithMessage(err, "Node: apiProfileUpdate: failed to bind json")
+		n.log.Debug(err)
+		api.AbortRequest(c, http.StatusBadRequest, "invalid_request_body", err)
```

Comment:
the same, choose 1: use api.AbortRequest, or do all your things right here

Reply:
Not sure what you mean :)
I chose to use `api.AbortRequest` so all places with `AbortWithStatusJSON` replaced with `api.AbortRequest`

Reply:
I mean that maybe it's good idea to put your debug log inside the function? DRY?

Reply:
> your debug log

I did not add this log, I am just trying to make less changes as possible, maybe this log here for some reason
> inside the function

Agree, I already did it, but noticed that 
sometimes it use [log.Debug](https://github.com/momentum-xyz/ubercontroller/blob/develop/universe/node/api_profile.go#L26)
sometimes [log.Error](https://github.com/momentum-xyz/ubercontroller/blob/develop/universe/node/api_profile.go#L75)
Looks like case with `log.Error` reasonable

Reply:
this log added by me, because we didn't have `api.AbortRequest` function, but now we have it, let's move log inside
about `log.Debug` and `log.Error` - it easy let's log important errors like 500 with `log.Error` and errors like 403 with `log.Debug`